images/stolonctl/entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 # -*- mode: sh; -*-
 
 # File: entrypoint.sh
-# Time-stamp: <2018-03-28 11:51:14>
+# Time-stamp: <2019-05-20 13:59:04>
```

Comment:
Is this metadata useful anywhere now that we store source code in a version control system?